README.md

```diff
@@ -84,27 +84,22 @@ Hence, a Dynatrace instance is required and the app must be bound to that Dynatr
 Use via `cds.requires.telemetry.kind = 'to-dynatrace'`.
 
 Required additional dependencies:
-- `@dynatrace/oneagent-sdk`
 - `@opentelemetry/exporter-trace-otlp-proto`
 - `@opentelemetry/exporter-metrics-otlp-proto`
 
-The necessary scope for exporting metrics (`metrics.ingest`) is not part of the standard `apitoken` and must be requested.
+The necessary scopes for exporting traces (`openTelemetryTrace.ingest`) and metrics (`metrics.ingest`) are not part of the standard `apitoken` and must be requested.
 This can only be done via binding to a "managed service instance", i.e., not a user-provided service instance.
```

Comment:
```suggestion
This can be done via parameterizing the binding to a "managed service instance" (i.e., not a user-provided service instance) as follows.
```